index.html

```diff
@@ -0,0 +1,525 @@
+          <img src="https://static01.nyt.com/images/2020/07/09/business/00CITYCAMERAS-GRID/00CITYCAMERAS-GRID-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale" alt="ad 7"
+           class="ad-image">
+          <small class="image-owner">Image by Cari Vander Yacht</small>
+          <p class="ad-text">Is Sex Dead for Parents?</p>
+          <p class="ad-date">July 9</p>
+        </div>
+        <!--8-->
+        <div class="ad">
+          <img src="https://static01.nyt.com/images/2020/07/09/business/09Techfix-illo/07Techfix-illo-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale" alt="ad 8"
+           class="ad-image">
+          <small class="image-owner">Glenn Harvey</small>
+          <p class="ad-text">How to Buy Tech That Lasts and Lasts</p>
+          <p class="ad-date">July 8</p>
+        </div>
+        <!--9-->
+        <div class="ad">
+          <img src="https://static01.nyt.com/images/2020/07/02/nyregion/00nyvirus-lines1/merlin_174168435_234ae7cd-aca1-44f7-94a5-d8b85e1c9bfb-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale" alt="ad 9"
+           class="ad-image">
+          <small class="image-owner">Ishita Jain</small>
+          <p class="ad-text">The True Story of My Marriage to a Pot</p>
+          <p class="ad-date">July 9</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="popular">
+      <h4> Most Popular</h4>
+      <hr>
+      <p>Dismemberment of Tech C.E.O. Fahim Saleh Looks Like ‘Professional Job’</p>
+      <hr>
+      <p>Opinion: Who Can Make Trump Miserable This Fall?</p>
+      <hr>
+      <p>How a Famous Harvard Professor Became a Target Over His Tweets</p>
+      <hr>
+      <p>Even if the Polls Are Really Off, Trump Is Still in Trouble</p>
+      <hr>
+      <p>The White House Called a News Conference. Trump Turned It Into a Meandering Monologue.</p>
+      <hr>
+      <p>Radio Host Loses Her Show Over English-Only Rant</p>
+      <hr>
+      <p>Comet NEOWISE: How to See It in Night Skies</p>
+      <hr>
+      <p>Bari Weiss Resigns From New York Times Opinion Post</p>
+      <hr>
+      <p>Trump Raises New Objections to Subpoena Seeking His Tax Returns</p>
+      <hr>
+      <p>Kansas Congressman Is Charged With Voter Fraud</p>
+    </div>
+  </div>
+
+  <footer id="nyt-links">
+    <hr>
+    <hr>
+    <section id="footer-header">
+      <img src="https://www.goyonder.it/wp-content/uploads/2019/09/2a9.f63.NYT-large-black-clear-background.png" alt="new-york-times" class="navbar-logo">
+      <span class="home-page">Go To Home Page >></span>
+    </section>
+    <section id="footer-links">
+      <div class="footer-list">
+        <h5>NEWS</h5>
+        <a>Home Page</a>
+        <a>World</a>
+        <a>U.S.</a>
+        <a>Politics</a>
+        <a>Election 2020</a>
+        <a>New York</a>
+        <a>Business</a>
+        <a>Tech</a>
+        <a>Science</a>
+        <a>Sports</a>
+        <a>Obituaries</a>
+        <a>Today's Paper</a>
+        <a>Corrections</a>
```

Comment:
- [ ] **[MANDATORY]** All of these should be displayed as links, but in your page they can not be clicked, to fix this, add the `<a href="#">` attribute